src/styles.jl

```diff
@@ -154,6 +154,19 @@ styles_numFmt_formatCode(wb::Workbook, numFmtId::Int) = styles_numFmt_formatCode
 
 const DATETIME_CODES = ["d", "m", "yy", "h", "s", "a/p", "am/pm"]
 
+function remove_formatting(code::AbstractString)
+    # this regex should cover all the formatting cases found here(colors/conditionals/quotes/spacing):
+    # https://support.office.com/en-us/article/create-or-delete-a-custom-number-format-78f2a361-936b-4c03-8772-09fab54be7f4
+    ignoredformatting = r"""
+        \[.{2,}?\]|
+        &quot;.+&quot;|
+        _.{1}|
+        \\.{1}|
+        \*.{1}
+        """ix
+    replace(code, ignoredformatting, "")
```

Comment:
Use `replace(code, ignoredformatting => "")` for Julia 1.0 compatibility. You'll probably need Compat.jl

Reply:
Please, don't put a dependency on Compat.
Don't worry about Julia 1.0 for now. I'll port this to Julia v1.0 once current dependencies work on v1.0 .

Reply:
Just curious: why you don't want to add a dependency on Compat.jl?

Reply:
Totally fine with me. I usually go with the approach of adopting the latest language syntax and using Compat to support older versions. It seems like every package author has a slightly different approach here.